testdata/exporter/app-slices/layers/app/metadata.toml

```diff
@@ -0,0 +1,11 @@
+[[processes]]
```

Comment:
This `metadata.toml` file in the `app` dir is a bit confusing, the one we actually respect lives in the `config` dir. I don't think this file was introduces in this PR but merely copied to a new location.

Reply:
@ekcasey I copied most of the fixture structure from an existing fixture.  If I don't need this, I'm more than happy to remove it.